### DISCUSSION TITLE
Turn off the sensor in the robot vacuum cleaner

### DISCUSSION THREAD

**Discussion by milkos86** [Hypfer/Valetudo#1753] [Q&A / Support] [answered]
Hello, tell me, using your firmware, you can disable the sensor in the robot vacuum cleaner forever?
The magnetic field sensor, it is also called the compass sensor.
Without disturbing the performance of the vacuum cleaner robot itself, with a connection to mihome
Robot vacuum cleaner xiaomi 1C

**Comment by Hypfer** [comment, marked as answer]
Quoting the first few words of the readme:

> Valetudo is a cloud replacement for vacuum robots enabling local-only operation. It is not a custom firmware.
> Because Valetudo is not a custom firmware, it cannot change anything about how the robot operates.

Literally the opposite of what you're asking for. It removes the cloud. It keeps the firmware as-is. Not vice-versa.
